models/kyberswap/avalanche_c/kyberswap_avalanche_c_schema.yml

```diff
@@ -23,7 +23,7 @@ models:
       - check_dex_seed:
           blockchain: avalanche_c
           project: kyberswap
-          version: dmm
+          version: elastic_2
```

Comment:
Do you have the corresponding seed added in this file:
https://github.com/duneanalytics/spellbook/blob/main/seeds/dex/trades/dex_trades_seed.csv

Reply:
I suggest we keep the `dmm`, and also add one new `check_dex_seed` test for `elastic_2`.

Reply:
@Hosuke `dmm` should be called `classic`, I've changed it to `version: classic`, pls check.